css/contact.css

```diff
@@ -0,0 +1,142 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* contact form */
+.wrapper {
+	width: var(--mobile-wrapper-width);
+}
+.form-input {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	/* padding: 20px; */
+}
+input[type="text"],
+[type="email"],
+select,
+textarea {
+	resize: vertical;
+	font-family: var(--main-font);
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	color: var(--color-onyx);
+	background-color: var(--color-white);
+	border: 1px solid #6c6c6c;
+	width: 100%;
+	border-radius: 5px;
+	padding: 10px;
+	margin: 5px 0;
+}
+.message {
+	min-height: 140px;
+	width: 100%;
+	border-radius: 10px;
+	border: none;
+	margin: 10px 0;
+	align-self: flex-start;
+}
+.button2 {
+	border-radius: 10px;
+	background-color: var(--button2-color);
+	padding: 10px 40px;
+	color: var(--color-white);
+	border: none;
+	min-width: 187px;
+	min-height: 50px;
+	margin: 20px;
+	font-weight: var(--font-weight-bold);
+	font-size: var(--font21);
+}
+.button2:hover {
+	background-color: var(--color-white);
+	color: var(--color-onyx);
+}
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+body {
+	background: var(--color-bluepurple);
+	color: var(--color-white);
+}
+h3 {
+	color: var(--color-white);
+	text-align: center;
+}
+p {
+	color: var(--color-white);
+	gap: 20px;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+	.contact-form-parent {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		text-align: left;
+		margin: 100px 150px;
```

Comment:
<img width="1473" alt="Screen Shot 2024-01-11 at 1 28 50 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/25f2f8d4-751c-4017-b02c-a77628bfa05c">


Since there's not a lot of content on this page, this container does not fill the height of the screen on desktop. 

- Try adding `min-height: 100vh;`
- Change the margin to padding `padding: 100px 150px`. This calculate that spacing within the container instead of pushing outside of it.